layouts/partials/tw/modal-takeover.html

```diff
@@ -0,0 +1,121 @@
+{{ $imagename := .filename }}
+{{ $hed := .hed }}
+{{ $dek := .dek }}
+{{ $link := .link }}
+
+<section
+  data-ga-category="modal:donate:takeover"
+  x-data="sticky"
+  x-show="isOpen"
+  x-transition:enter="transition ease-out duration-500"
+  x-transition:enter-start="opacity-0 transform translate-x-8"
+  x-transition:enter-end="opacity-100 transform translate-x-0"
+  x-transition:leave="transition ease-in duration-500"
+  x-transition:leave-start="opacity-100 transform translate-x-0"
+  x-transition:leave-end="opacity-0 transform translate-x-8"
+  @open-sticky.window="show()"
+  @close-sticky.window="close()"
+  data-ga-category="modal:sticky"
+  @keydown.escape="close"
+  x-cloak
+  aria-modal="true"
+  aria-label="Contribute"
+  class="fixed bottom-0 left-0 z-30 flex flex-wrap md:left-auto md:top-0 md:right-0 md:h-full md:w-1/3"
+>
+  <button
+    class="absolute right-3 -top-10 flex rounded-xl bg-white text-white md:top-4 md:right-auto md:left-5"
+    type="button"
+    data-ga-action="modal:takeover:dismiss"
+    @keydown.escape="isOpen = false"
+    aria-label="close"
+    @click="close"
+    x-ref="close"
+  >
+    <svg class="my-auto box-content h-6 w-6 fill-black">
+      {{ partial "helper/svg.html" "close" }}
+    </svg>
+  </button>
+  <a
+    href="{{ $link }}"
+    data-sf-campaign="701Dn000000YqKfIAK"
+    class="bottom-0 flex flex-wrap overflow-y-scroll ring-red focus:outline-none focus:ring-2 md:left-auto md:top-0 md:h-full"
+    :class="{'left-full': !isOpen, 'overflow-y-hidden': !isOpen}"
+  >
+    <header
+      class="z-20 flex-1 self-start overflow-visible font-serif font-bold text-white sm:min-w-full md:flex md:h-0 md:items-end md:bg-black"
+    >
+      <h2
+        class="bg-red p-4 text-xl md:mr-9 md:-mb-28 md:p-10 md:text-2xl lg:text-3xl xl:text-4xl"
+      >
+        {{ $hed }}
+      </h2>
+    </header>
+    <span class="c-red-8 -order-1 self-end sm:order-1 sm:self-start md:mt-28">
+      <svg
+        class="fill-red-8 sm:origin-center sm:scale-[(-1,_1)]"
+        width="16"
+        height="22"
+      >
```

Comment:
Maybe add a comment like `<!-- triangle -->` so it's clear that it's the little fold out triangle.